doc/tutorials/tarball_installation.rst

```diff
@@ -0,0 +1,248 @@
+files. You might need to modify the commands for your environment.
+
+Prerequisites
+*************
+
+* A development computer configured according to the :ref:`gsg` (for example, it
+  has an Ubuntu OS, ACRN dependencies, and ACRN hypervisor and kernel source
+  code).
+
+Build the Tarball Files
+***************************
+
+#. On your development computer, navigate to the ``acrn-hypervisor`` source code
+   and build the ACRN hypervisor tarball files. Be sure to replace
+   ``<board.xml>`` and ``<scenario.xml>`` with the paths to your board
+   configuration file and scenario configuration file.
+
+   .. code-block:: bash
+
+      cd ~/acrn-work/acrn-hypervisor
+      make -j $(nproc) BOARD=<board.xml> SCENARIO=<scenario.xml>
+      make targz-pkg
+
+#. Navigate to the ``acrn-kernel`` source code and build the ACRN kernel tarball
+   files for the Service VM:
+
+   .. code-block:: bash
+
+      cd ~/acrn-work/acrn-kernel
+      cp kernel_config_service_vm .config
+      make olddefconfig
+      make -j $(nproc) targz-pkg
+
+#. Copy all the necessary files generated on the development computer to the
+   target system by USB disk as follows. Modify the file names in the following
+   commands to match your files.
+
+   a. Insert the USB disk into the development computer and run these commands:
+
+      .. code-block:: bash
+
+         disk="/media/$USER/"$(ls /media/$USER)
+         cp linux-5.10.65-acrn-service-vm-x86.tar.gz "$disk"/
+         cp ~/acrn-work/acrn-hypervisor/build/hypervisor/acrn.bin "$disk"/
+         cp ~/acrn-work/acpica-unix-20210105/generate/unix/bin/iasl "$disk"/
+         cp ~/acrn-work/acrn-hypervisor/build/acrn-2.7-unstable.tar.gz "$disk"/
+         sync && sudo umount "$disk"/
+
+      .. note:: The :file:`serial.conf` is only generated if non-standard
+         vUARTs (not COM1-COM4)
+         are configured for the Service VM in the scenario XML file.
+         Please copy the ``serial.conf`` file using::
+            
+            cp ~/acrn-work/acrn-hypervisor/build/hypervisor/serial.conf "$disk"/
+
+   #. Insert the USB disk you just used into the target system and run these
+      commands to copy the tar files locally:
+
+      .. code-block:: bash
+
+         disk="/media/$USER/"$(ls /media/$USER)
+         cp "$disk"/linux-5.10.65-acrn-service-vm-x86.tar.gz ~/acrn-work
+         cp "$disk"/acrn-2.7-unstable.tar.gz ~/acrn-work
+
+   #. Extract the Service VM files onto the target system:
+
+      .. code-block:: bash
+
+         cd ~/acrn-work
+         sudo tar -zxvf linux-5.10.65-acrn-service-vm-x86.tar.gz -C / --keep-directory-symlink
+
+   #. Extract the ACRN tools and images:
+
+      .. code-block:: bash
+
+         sudo tar -zxvf acrn-2.7-unstable.tar.gz -C / --keep-directory-symlink
+
+   #. Copy a few additional ACRN files to the expected locations:
+
+      .. code-block:: bash
+
+         sudo mkdir -p /boot/acrn/
+         sudo cp "$disk"/acrn.bin /boot/acrn
```

Comment:
```suggestion
         sudo cp /usr/lib64/acrn/acrn.<board>.<scneario>.bin /boot/acrn/acrn.bin
```

Reply:
I would move this line out of this block because this new command does not take the source file from the USB stick (so it's the odd one out in the mix). I would put it in the section below and add a note telling the user he needs to replace `<board>` and `<scenario>` with his own values.

Reply:
@gvancuts The end of your suggested command is `/boot/acrn/acrn.bin` -- does this rename the bin file?

Reply:
Yes, it does. It's on purpose so that it matches the name used in the Grub example below. We can also choose not to rename it but in that case we need to update the Grub section below (`multiboot2` line) to match the name used here.

Reply:
I chose to keep the original name and change the Grub example accordingly. Also, there were two sets of copy commands, one before extracting the tar files and one after extracting the tar files. I consolidated all of the copy commands and put them before extracting the tar files. Please review.